Move docs to mkdocs base (#134)
Ismoh/NoitaMP: docs/index.md

NoitaMP is a mod for [Noita](https://noitagame.com/) to be able to share your feelings while playing Noita with your friends. It's a multiplayer mod :rage1: View it on [GitHub](https://github.com/Ismoh/NoitaMP).

> ## TL;DR

Reviewer: ```suggestion
## TL;DR / Installation
```

**Currently work in progress and not working unless there is a GitHub release available!**

> If you want to get notifications, [join](https://discord.gg/DhMurdcw4k) my discord server. There is a MEE6 role assigment for NoitaMP in #noita-modding!

Reviewer: ```suggestion
If you want to get notifications, [join](https://discord.gg/DhMurdcw4k) my discord server. There is a carl.gg role assigment for NoitaMP in #noita-modding!
```

> If you want to get notifications, [join](https://discord.gg/DhMurdcw4k) my discord server. There is a carl.gg role assigment for NoitaMP in #noita-modding!

Reviewer: ```suggestion
If you want to get notifications, [join](https://discord.gg/DhMurdcw4k) my discord server. There is a carl.gg role assigment for NoitaMP in #noitamp-chat!
```
